.github/workflows/build_all.yml

```diff
@@ -63,6 +63,6 @@ jobs:
       distribution: ubuntu-22.04
       archive-name: ubuntu_22_04_tests
       commands: |
-        cargo build --bins
-        cargo test --workspace -- --nocapture --skip test_mysql --skip test_pgsql --skip test_redis
+        cargo build --release --bins
+        cargo test --release --workspace -- --nocapture --skip test_mysql --skip test_pgsql --skip test_redis
```

Comment:
please remove these added `--release` flags in this file. This file is about running tests, so we don't want `release` flag to speed-up CI

Reply:
For now, I increase `RUST_MIN_STACK` to avoid stack size overflow during the tests in Debug.

Reply:
fixed in https://github.com/Cosmian/kms/pull/199